config/seckit.settings.yml

```diff
@@ -0,0 +1,56 @@
+seckit_xss:
+  csp:
+    checkbox: true
+    vendor-prefix:
+      x: false
+      webkit: false
+    report-only: true
+    default-src: ''
+    script-src: '''self'' ''unsafe-inline'' https://fonts.googleapis.com https://www.gstatic.com https://www.google.com https://www.googletagmanager.com https://www.google-analytics.com https://cdn.jsdelivr.net'
+    object-src: ''
+    img-src: '''self'' data:'
+    media-src: ''
+    frame-src: ''
+    frame-ancestors: ''
+    child-src: ''
+    font-src: 'https://fonts.gstatic.com'
+    connect-src: '''self'' https://www.google-analytics.com https://stats.g.doubleclick.net'
+    report-uri: /report-csp-violation
+    upgrade-req: false
+    policy-uri: ''
+    style-src: '''self'' https://fonts.googleapis.com'
```

Comment:
We seem to have a non-HTTPS reference in our codebase. I'd opt to fix the implementation rather than loosen this CSP rule.

```
[Report Only] Refused to load the stylesheet 'http://fonts.googleapis.com/css?family=Roboto:ital,wght@0,300%3B0,400%3B0,500%3B0,700%3B0,900%3B1,400%3B1,700&display=swap' because it violates the following Content Security Policy directive: "style-src 'self' https://fonts.googleapis.com". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.
```